src/tests/Interop/Swift/SwiftErrorHandling/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+project(SwiftErrorHandling)
+include ("${CLR_INTEROP_TEST_ROOT}/Interop.cmake")
+
+set(SOURCE SwiftErrorHandling)
+
+add_custom_target(${SOURCE} ALL
+    COMMAND xcrun swiftc -emit-library ${CMAKE_CURRENT_SOURCE_DIR}/${SOURCE}.swift -o ${CMAKE_CURRENT_BINARY_DIR}/lib${SOURCE}.dylib
+    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${SOURCE}.swift
+    COMMENT "Generating ${SOURCE} library"
+)
```

Comment:
Is this the appropriate way to do this? Seems like CMake has first class support for Swift - https://github.com/compnerd/swift-cmake-examples/tree/main. Is there a reason to create a custom target?

Reply:
CMake currently supports Swift with Ninja and Xcode generators. By default, we use `Unix Makefiles` generator (on Ubuntu and OSX), so we decided to follow the example from https://github.com/dotnet/runtime/blob/3b61ad9e3acb6302fd2fcef4947373475a0dcc29/src/native/libs/System.Security.Cryptography.Native.Apple/CMakeLists.txt#L54.

Reply:
I see. I'd like to hear @jkoritzinsky's opinion on this approach.

Reply:
I think this is fine.

If we were to switch to using Ninja by default, I'd like to move to using the built-in support when using Ninja, but since that's not in any plans, I don't think it's worth the effort.